swift/internal/compiling.bzl

```diff
@@ -2879,27 +2919,6 @@ def _declare_multiple_outputs_and_write_output_file_map(
         derived_files_output_file_map = derived_files_output_map_file,
     )
 
-def _declare_validated_generated_header(actions, generated_header_name):
-    """Validates and declares the explicitly named generated header.
-
-    If the file does not have a `.h` extension, the build will fail.
-
-    Args:
-        actions: The context's `actions` object.
-        generated_header_name: The desired name of the generated header.
-
-    Returns:
-        A `File` that should be used as the output for the generated header.
-    """
-    extension = paths.split_extension(generated_header_name)[1]
-    if extension != ".h":
-        fail(
-            "The generated header for a Swift module must have a '.h' " +
-            "extension (got '{}').".format(generated_header_name),
-        )
-
-    return actions.declare_file(generated_header_name)
```

Comment:
@tymurmustafaiev out of curiosity, why was this validation removed?

Reply:
![Screenshot 2023-07-20 at 15 10 43](https://github.com/revolut-mobile/rules_swift/assets/17967811/530edf6e-52a5-433c-bb78-a288a4b20111)

🤦 

Ok, thanks